keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -899,13 +990,19 @@ No change from existing behavior of the Job controller.
 
 #### What are other known failure modes?
 
+There are no other failure modes.
+
 #### What steps should be taken if SLOs are not being met to determine the problem?
 
+No, SLI/SLO do not include time taking to create new pods if existing ones are terminated. 
```

Comment:
How about suspending the jobs that are experiencing increased failures due to the feature being off, see my other comment in rollout fail question.

Reply:
+1

Reply:
In terms of the job controller, I am a bit confused by what we mean by "suspending the jobs"  Are you saying that one could `suspend:true` the jobs in question?

Reply:
Updated.